Quarkus-native failure "No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime."

**Describe the bug**
As of https://github.com/oracle/graal/commit/39d7ff72e9b02d24a333d176c9dc3166afabd6a3 
Multiple integration tests fail with "No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime."

(See https://github.com/oracle/graal/runs/1196947890?check_suite_focus=true#step:17:1022)

**Expected behavior**
Integration tests should pass with latest graal.

**Actual behavior**
Multiple integration tests fail after https://github.com/oracle/graal/commit/39d7ff72e9b02d24a333d176c9dc3166afabd6a3


(See https://github.com/oracle/graal/actions/runs/283538106)

**To Reproduce**

1. Build latest graal master or download latest dev build from https://github.com/graalvm/graalvm-ce-dev-builds/releases/
2. Install native-image `gu install native-image`
3. Export `GRAALVM_HOME` to point to the graal you built or downloaded in step 1.
4. Run the integration tests with `-Pnative` e.g.
```
$ ./mvnw --settings .github/mvn-settings.xml -Dnative -Dnative.skip.surefire -Dquarkus.native.additional-build-args='--trace-object-instantiation=java.awt.color.ICC_ColorSpace' -pl integration-tests/main verify
```

**Environment:**
 - Output of `uname -a` or `ver`: Linux 5.8.7-200.fc32.x86_64 #1 SMP Mon Sep 7 15:26:10 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: openjdk version "11.0.8" 2020-07-14
 - GraalVM version (if different from Java):  GraalVM CE 20.3.0-dev-20201003_0219
 - Quarkus version or git rev: master
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)

**Additional context**
Output of `./mvnw --settings .github/mvn-settings.xml -Dnative -Dnative.skip.surefire -Dquarkus.native.additional-build-args='--trace-object-instantiation=java.awt.color.ICC_ColorSpace' -pl integration-tests/main verify`
```
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] /graalvm-ce-java11-20.3.0-dev/bin/native-image -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-DCoordinatorEnvironmentBean.transactionStatusManagerEnable=false -J-Dcom.sun.xml.bind.v2.bytecode.ClassTailor.noOptimize=true -J-Drx.unsafe-disable=true -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=1 -J-Duser.language=en -J-Dfile.encoding=UTF-8 --trace-object-instantiation=java.awt.color.ICC_ColorSpace --initialize-at-run-time=javax.imageio.ImageTypeSpecifier -H:IncludeResources=test-resources/.*\.txt --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+JNI -jar quarkus-integration-test-main-999-SNAPSHOT-runner.jar -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:+AddAllCharsets -H:EnableURLProtocols=http -H:NativeLinkerOption=-no-pie --no-server -H:-UseServiceLoaderFeature -H:+StackTrace quarkus-integration-test-main-999-SNAPSHOT-runner
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]    classlist:   7,715.50 ms,  1.69 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]        (cap):     751.04 ms,  1.69 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]        setup:   2,769.94 ms,  1.69 GB
00:21:27,458 INFO  [org.hib.val.int.uti.Version] HV000001: Hibernate Validator 6.1.6.Final
00:21:27,609 INFO  [org.hib.Version] HHH000412: Hibernate ORM core version 5.4.22.Final
00:21:27,613 INFO  [org.hib.ann.com.Version] HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
00:21:27,649 INFO  [org.hib.dia.Dialect] HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL95Dialect
00:21:27,828 INFO  [io.und.web.jsr] UT026003: Adding annotated server endpoint class io.quarkus.it.websocket.RecodingSocket for path /recoder
00:21:27,830 INFO  [io.und.web.jsr] UT026003: Adding annotated server endpoint class io.quarkus.it.websocket.WebSocketOpenEndpoint for path /wsopen
00:21:27,830 INFO  [io.und.web.jsr] UT026004: Adding annotated client endpoint class io.quarkus.it.websocket.CodingClient
00:21:27,831 INFO  [io.und.web.jsr] UT026003: Adding annotated server endpoint class io.quarkus.it.websocket.EchoSocket for path /echo
00:22:06,504 INFO  [org.jbo.threads] JBoss Threads version 3.1.1.Final
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]     (clinit):   1,691.83 ms,  6.33 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]   (typeflow):  38,714.85 ms,  6.33 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]    (objects):  44,000.36 ms,  6.33 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]   (features):   2,431.97 ms,  6.33 GB
[quarkus-integration-test-main-999-SNAPSHOT-runner:2149929]     analysis:  91,953.50 ms,  6.33 GB
Error: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1876)
Call path from entry point to com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(): 
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1869)
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.getType(JPEGImageReader.java:1847)
        at com.sun.imageio.plugins.jpeg.ImageTypeIterator.hasNext(JPEGImageReader.java:1798)
        at java.util.AbstractSet.hashCode(AbstractSet.java:121)
        at java.util.HashMap.hash(HashMap.java:339)
        at java.util.HashMap.get(HashMap.java:552)
        at com.oracle.svm.core.jdk.Target_java_nio_charset_Charset.lookup(CharsetSubstitutions.java:74)
        at java.nio.charset.Charset.isSupported(Charset.java:503)
        at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_VA_LIST:Ljava_nio_charset_Charset_2_0002eisSupported_00028Ljava_lang_String_2_00029Z(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:1042)
Call path from entry point to com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(BufferedImage, ImageReadParam): 
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:984)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.readInternal(JPEGImageReader.java:1154)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.read(JPEGImageReader.java:1110)
        at javax.imageio.ImageIO.read(ImageIO.java:1468)
        at javax.imageio.ImageIO.read(ImageIO.java:1420)
        at io.quarkus.it.corestuff.ImageIOSupport.doGet(ImageIOSupport.java:18)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:503)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:590)
        at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
        at io.undertow.server.handlers.PathHandler.handleRequest(PathHandler.java:91)
        at io.quarkus.undertow.runtime.UndertowDeploymentRecorder$1.handleRequest(UndertowDeploymentRecorder.java:114)
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:290)
        at io.undertow.servlet.spec.AsyncContextImpl$TimeoutTask$1$1.run(AsyncContextImpl.java:515)
        at java.lang.Thread.run(Thread.java:834)
        at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
        at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
        at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

com.oracle.svm.core.util.UserError$UserException: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1876)
Call path from entry point to com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(): 
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1869)
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.getType(JPEGImageReader.java:1847)
        at com.sun.imageio.plugins.jpeg.ImageTypeIterator.hasNext(JPEGImageReader.java:1798)
        at java.util.AbstractSet.hashCode(AbstractSet.java:121)
        at java.util.HashMap.hash(HashMap.java:339)
        at java.util.HashMap.get(HashMap.java:552)
        at com.oracle.svm.core.jdk.Target_java_nio_charset_Charset.lookup(CharsetSubstitutions.java:74)
        at java.nio.charset.Charset.isSupported(Charset.java:503)
        at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_VA_LIST:Ljava_nio_charset_Charset_2_0002eisSupported_00028Ljava_lang_String_2_00029Z(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:1042)
Call path from entry point to com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(BufferedImage, ImageReadParam): 
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:984)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.readInternal(JPEGImageReader.java:1154)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.read(JPEGImageReader.java:1110)
        at javax.imageio.ImageIO.read(ImageIO.java:1468)
        at javax.imageio.ImageIO.read(ImageIO.java:1420)
        at io.quarkus.it.corestuff.ImageIOSupport.doGet(ImageIOSupport.java:18)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:503)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:590)
        at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
        at io.undertow.server.handlers.PathHandler.handleRequest(PathHandler.java:91)
        at io.quarkus.undertow.runtime.UndertowDeploymentRecorder$1.handleRequest(UndertowDeploymentRecorder.java:114)
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:290)
        at io.undertow.servlet.spec.AsyncContextImpl$TimeoutTask$1$1.run(AsyncContextImpl.java:515)
        at java.lang.Thread.run(Thread.java:834)
        at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
        at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
        at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

        at com.oracle.svm.core.util.UserError.abort(UserError.java:82)
        at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:217)
        at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:770)
        at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:560)
        at com.oracle.svm.hosted.NativeImageGenerator.lambda$run$0(NativeImageGenerator.java:473)
        at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Unsupported features in 2 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1876)
Call path from entry point to com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(): 
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.produce(JPEGImageReader.java:1869)
        at com.sun.imageio.plugins.jpeg.ImageTypeProducer.getType(JPEGImageReader.java:1847)
        at com.sun.imageio.plugins.jpeg.ImageTypeIterator.hasNext(JPEGImageReader.java:1798)
        at java.util.AbstractSet.hashCode(AbstractSet.java:121)
        at java.util.HashMap.hash(HashMap.java:339)
        at java.util.HashMap.get(HashMap.java:552)
        at com.oracle.svm.core.jdk.Target_java_nio_charset_Charset.lookup(CharsetSubstitutions.java:74)
        at java.nio.charset.Charset.isSupported(Charset.java:503)
        at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_VA_LIST:Ljava_nio_charset_Charset_2_0002eisSupported_00028Ljava_lang_String_2_00029Z(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.

Trace: 
        at parsing com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:1042)
Call path from entry point to com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(BufferedImage, ImageReadParam): 
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.checkColorConversion(JPEGImageReader.java:984)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.readInternal(JPEGImageReader.java:1154)
        at com.sun.imageio.plugins.jpeg.JPEGImageReader.read(JPEGImageReader.java:1110)
        at javax.imageio.ImageIO.read(ImageIO.java:1468)
        at javax.imageio.ImageIO.read(ImageIO.java:1420)
        at io.quarkus.it.corestuff.ImageIOSupport.doGet(ImageIOSupport.java:18)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:503)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:590)
        at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
        at io.undertow.server.handlers.PathHandler.handleRequest(PathHandler.java:91)
        at io.quarkus.undertow.runtime.UndertowDeploymentRecorder$1.handleRequest(UndertowDeploymentRecorder.java:114)
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:290)
        at io.undertow.servlet.spec.AsyncContextImpl$TimeoutTask$1$1.run(AsyncContextImpl.java:515)
        at java.lang.Thread.run(Thread.java:834)
        at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
        at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
        at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

        at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:129)
        at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:767)
        ... 8 more
Error: Image build request failed with exit status 1

```

> This does work, any idea why this is happening though?

It appears that: 
> These classes (java.awt.color.ColorSpace and java.awt.color.ICC_Profile) contain standard color profile caches which may not be loaded yet
see: https://github.com/oracle/graal/commit/39d7ff72e9b02d24a333d176c9dc3166afabd6a3#diff-16fa1354935120a447ef7e24561ec3c1R58

Now regarding how I came with the solution/work around.

If you build the `native-image` without any flags you will get the an error message asking you to use `--trace-object-instantiation` to identify the root cause.

If you add `--trace-object-instantiation=java.awt.color.ICC_ColorSpace` it will show traces of where the issue originates from.

So in my tests (see Additional Context in https://github.com/quarkusio/quarkus/issues/12535#issue-715173991) I got:
```
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the com.sun.imageio.plugins.jpeg.JPEG$JCS class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at com.sun.imageio.plugins.jpeg.JPEG$JCS.<clinit>(JPEG.java:208)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.
```

which indicates that the root cause is the initialization of `com.sun.imageio.plugins.jpeg.JPEG$JCS`  so adding this to `--initialize-at-run-time` as indicated by the error message makes that error go away.

Similarly for `javax.imageio.ImageTypeSpecifier` I got:
```
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of java.awt.color.ICC_ColorSpace are allowed in the image heap as this class should be initialized at image runtime. Object has been initialized by the javax.imageio.ImageTypeSpecifier class initializer with a trace: 
        at java.awt.color.ICC_ColorSpace.<init>(ICC_ColorSpace.java:112)
        at java.awt.color.ColorSpace.getInstance(ColorSpace.java:299)
        at javax.imageio.ImageTypeSpecifier.<clinit>(ImageTypeSpecifier.java:73)
.  To fix the issue mark java.awt.color.ICC_ColorSpace for build-time initialization with --initialize-at-build-time=java.awt.color.ICC_ColorSpace or use the the information from the trace to find the culprit and --initialize-at-run-time=<culprit> to prevent its instantiation.
```

HTH

--- a/core/deployment/src/main/java/io/quarkus/deployment/ImageIOProcessor.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/ImageIOProcessor.java
@@ -1,0 +1,21 @@
+package io.quarkus.deployment;
+
+import java.util.Arrays;
+import java.util.List;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
+import io.quarkus.deployment.pkg.steps.NativeBuild;
+
+public class ImageIOProcessor {
+
+    @BuildStep(onlyIf = NativeBuild.class)
+    List<RuntimeInitializedClassBuildItem> runtimeInitImageIOClasses() {
+        // The following classes hold instances of java.awt.color.ICC_ColorSpace that are not allowed in the image
+        // heap as this class should be initialized at image runtime
+        // (See https://github.com/quarkusio/quarkus/issues/12535)
+        return Arrays.asList(
+                new RuntimeInitializedClassBuildItem("javax.imageio.ImageTypeSpecifier"),
+                new RuntimeInitializedClassBuildItem("com.sun.imageio.plugins.jpeg.JPEG$JCS"));
+    }
+}

